src/main/java/org/tron/core/services/http/Util.java

```diff
@@ -16,18 +43,315 @@ public static String printErrorMsg(Exception e) {
+            any = Any.pack(witnessUpdateContractBuilder.build());
+            break;
+          case "ParticipateAssetIssueContract":
+            ParticipateAssetIssueContract.Builder participateAssetIssueContractBuilder = ParticipateAssetIssueContract
+                .newBuilder();
+            JsonFormat.merge(parameter.getJSONObject("value").toJSONString(),
+                participateAssetIssueContractBuilder);
+            any = Any.pack(participateAssetIssueContractBuilder.build());
+            break;
+          case "AccountUpdateContract":
+            AccountUpdateContract.Builder accountUpdateContractBuilder = AccountUpdateContract
+                .newBuilder();
+            JsonFormat.merge(parameter.getJSONObject("value").toJSONString(),
+                accountUpdateContractBuilder);
+            any = Any.pack(accountUpdateContractBuilder.build());
+            break;
+          case "FreezeBalanceContract":
+            FreezeBalanceContract.Builder freezeBalanceContractBuilder = FreezeBalanceContract
+                .newBuilder();
+            JsonFormat.merge(parameter.getJSONObject("value").toJSONString(),
+                freezeBalanceContractBuilder);
+            any = Any.pack(freezeBalanceContractBuilder.build());
+            break;
+          case "UnfreezeBalanceContract":
+            UnfreezeBalanceContract.Builder unfreezeBalanceContractBuilder = UnfreezeBalanceContract
+                .newBuilder();
+            JsonFormat.merge(parameter.getJSONObject("value").toJSONString(),
+                unfreezeBalanceContractBuilder);
+            any = Any.pack(unfreezeBalanceContractBuilder.build());
+            break;
+          case "UnfreezeAssetContract":
+            UnfreezeAssetContract.Builder unfreezeAssetContractBuilder = UnfreezeAssetContract
+                .newBuilder();
+            JsonFormat.merge(parameter.getJSONObject("value").toJSONString(),
+                unfreezeAssetContractBuilder);
+            any = Any.pack(unfreezeAssetContractBuilder.build());
+            break;
+          case "WithdrawBalanceContract":
+            WithdrawBalanceContract.Builder withdrawBalanceContractBuilder = WithdrawBalanceContract
+                .newBuilder();
+            JsonFormat.merge(parameter.getJSONObject("value").toJSONString(),
+                withdrawBalanceContractBuilder);
+            any = Any.pack(withdrawBalanceContractBuilder.build());
+            break;
+          case "UpdateAssetContract":
+            UpdateAssetContract.Builder updateAssetContractBuilder = UpdateAssetContract
+                .newBuilder();
+            JsonFormat
+                .merge(parameter.getJSONObject("value").toJSONString(), updateAssetContractBuilder);
+            any = Any.pack(updateAssetContractBuilder.build());
+            break;
+          // todo add other contract
+          default:
+        }
+        String value = ByteArray.toHexString(any.getValue().toByteArray());
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: A "NullPointerException" could be thrown; "any" is nullable here. ([squid:S2259](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2259))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1125&issues=AWRuxZPJl6vpliCeCUMR&open=AWRuxZPJl6vpliCeCUMR)

[ISSUE_KEY]: AWRuxZPJl6vpliCeCUMR